src/circuit/dialer.js

```diff
@@ -0,0 +1,186 @@
+'use strict'
+
+require('safe-buffer')
```

Comment:
I believe you need to `const Buffer = require('safe-buffer').Buffer` for it to work